dept44-example/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
```

Comment:
Remove version definition for testcontainers-dependencies (these are managed by SB now)?

Reply:
I guess we could lift the version to 6.0.0 as well? 
https://mvnrepository.com/artifact/io.github.git-commit-id/git-commit-id-maven-plugin

Dependebot will probably take care of this, but while you're at it... :-)